maintenance/projects/js-toolkit/packages/generator-liferay-js/src/target-vanilla-portlet/vanilla-portlet.js

```diff
@@ -55,8 +55,9 @@ export default class extends Generator {
 		// Configure build
 
 		if (useBabel) {
-			pkgJson.addDevDependency('babel-cli', '6.26.0');
-			pkgJson.addDevDependency('babel-preset-env', '1.7.0');
+			pkgJson.addDevDependency('@babel/cli', '^7.7.5');
```

Comment:
@jonmak08 Why do you need to change the vanilla JS portlet too? Isn't it enough with changing the react one? :thinking:

Reply:
And the same question for the shared bundle target...

Reply:
I thought if we were going to upgrade babel, we wanted to upgrade it everywhere in the toolkit, but if not, we can just upgrade it in only the react portlet templates.

Reply:
I think the main concern @izaera had was with the bundler that is invoked as part of the build script resulting from the generator:

```json
"build": "npm run copy-assets && liferay-npm-bundler"
```

In its latest released version (I think) it [still uses babel 6](https://github.com/liferay/liferay-js-toolkit/blob/master/packages/liferay-npm-bundler/package.json#L18) and it's this combination the one that both, has us worried that it might not work as expected and that changing it there might cause issues elsewhere in the toolchain.

Reply:
Well, we may change it here too, test it, and see what happens. As long as we pass it through QA and it works...

@wincent [said](https://github.com/liferay/liferay-frontend-projects/pull/466#issuecomment-806450947) we already mix babel 7 and 6 in liferay-portal and nothing happens, so I guess it is safer than it seems.

My previous comments may be a bit misleading because I was assuming (without being too much aware of it) that we needed to change babel inside the bundler too but I realized yesterday (due to a comment of @jbalsas in slack) that this change only affects user's projects, so it is very possible that some people out there may be already using babel 7 to transpile and the bundler to deploy. We don't know, but I wouldn't find it strange given that nobody is complaining about babel 6 since long ago (note that another reason could be that nobody is using the toolkit too :sweat_smile: ).

So I would say, let's change it, test it to make sure nothing breaks and release.

@john-co is having problems setting the old QA code because it broke when we migrated to liferay-frontend-projects, so I will have a look at it today and see what can be done to fix/change it.

Reply:
> [...] so it is very possible that some people out there may be already using babel 7 to transpile and the bundler to deploy [...]

We know @ethib137 was, that's how we got here 😉

Reply:
I use it all the time and tell everyone to use it. 😉